rust-toolchain.toml

```diff
@@ -0,0 +1,7 @@
+[toolchain]
+channel = "nightly"
```

Comment:
What nightly features do you need?

Reply:
Nightly is needed for the [asm-const](https://doc.rust-lang.org/unstable-book/language-features/asm-const.html) feature. In retrospect, I may be able to circumvent this as inline assembly is just implemented with format strings.